content/en/service_management/incident_management/investigate/timeline.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Timeline
+further_reading:
+- link: "/service_management/incident_management/investigate/"
+  tag: "Documentation"
+  text: "Investigate incidents"
+---
+
+## Overview
+
+{{< img src="/service_management/incidents/investigate/timeline/timeline_tab.png" alt="Example incident showing the Timeline tab" style="width:100%;" >}}
+
+The Incident Timeline is the primary source of information for the work done during an incident. As actions are performed, new cells are added to the timeline in chronological order to capture the changes made, the person who made the change, and the time the changes were made.
+
+By default, timeline cells are sorted in `oldest first` order, but this can be changed to `newest first` using the button at the top of the timeline.
+
+## Content types
+
+Each cell has its own content type that indicates the kind of information the cell contains:
+
+|  Content type      | Description                                                                                            |
+| ------------------ | ------------------------------------------------------------------------------------------------------ |
+| Responder note     | A note manually written by an incident responder. Responder notes have the following sub-types:<br>- *Graph*: The responder note contains one or more Datadog graphs<br>- *Link*: The responder note contains a hyperlink<br>- *Code*: The responder note contains text wrapped in Markdown syntax for code blocks
```

Comment:
Does a responder note always have to be of one of the subtypes, or is there just a "plain" responder note type?

Reply:
You can have a plain note, within a note cell you can add graphs and links. If you don't add anything, it'd just be a responder note.